guidelines/coding-guidelines.md

```diff
@@ -11,3 +11,21 @@ For public facing interfaces we favor read-only collections and enumerables. For
 * Avoid using `System.Linq`
 * Avoid using `foreach` over collections that do not have a struct enumerator
 
+## Don't use string interpolation
+
+By using string interpolation it is impossible for logging frameworks that support structure logging to distinguish values within the string. We must use the normal argument based overload so logging frameworks like Serilog can capture structured data.
```

Comment:
You're not missing it @SimonCropp They have a custom format. Additionally, there was a post describing that even if they knew how to match the variable, they wouldn't do it as Serilog follows "make implicit explicit" and they want user to name properties.